lms/static/sass/course/layout/_courseware_preview.scss

```diff
@@ -47,47 +47,57 @@
     font-size: 90%;
   }
 }
-.exam-timer {
-  line-height: 56px;
-  background-color: #e5eaec;
-  padding-left: 42px;
-  padding-right: 32px;
-  border-left: 4px solid #14a6ea;
-  margin: 0 auto;
-  color: #4e575b;
-  font-size: 14px;
-  a {
-    color: #0979ba;
+.proctored_exam_status {
+  // STATE: Fixed to viewport (used for scrolling)
+  &.is-fixed {
+    @extend %ui-depth4;
+    box-shadow: 0 3px 3px $shadow-d1;
+    position: fixed;
+    top: 0;
+    width: 100%;
   }
-  span.pull-right {
-    color: #646161;
+  .exam-timer {
```

Comment:
NOTE:
`.exam-timer` is also in the `proctored_exam_status ` div so i just moved the `.exam-timer` sass to the `proctored_exam_status` div